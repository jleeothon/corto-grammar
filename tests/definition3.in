Point a: x=3, y=4
