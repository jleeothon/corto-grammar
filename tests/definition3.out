(program (statement (definition (declaration (typeLabel (identifier Point)) a) : (initializerList (initializerElem (initializerValue (identifier x)) = (initializerValue (literal 3))) , (initializerElem (initializerValue (identifier y)) = (initializerValue (literal 4))))) \n))
